Symfony/CS/Tests/Fixer/Contrib/DirConstantFixerTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\Contrib;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Vladimir Reznichenko <kalessil@gmail.com>
+ */
+class DirConstantFixerTest extends AbstractFixerTestBase
+{
+    protected function assertTokens(Tokens $expectedTokens, Tokens $tokens)
+    {
+        // tokens set and content can differ, so rely on produced code identity only
```

Comment:
This is totally wrong. It is **forbidden** to produce a corrupted collection. Following fixers will work on the collection, so it MUST be valid.